editor/script_editor_debugger.h

```diff
@@ -66,6 +66,7 @@ class ScriptEditorDebugger : public Control {
 	enum ItemMenu {
 		ITEM_MENU_COPY_ERROR,
 		ITEM_MENU_SAVE_REMOTE_NODE,
+		ITEM_MENU_COPY_NODE_PATH
```

Comment:
Please add a comma at the end, so that if another constant is added, we don't get a diff on this line.

Reply:
fixed